### ISSUE TITLE
cisco_nxos_show_vlan does not show ports, thank god cisco_ios_show_vlan works

### ISSUE THREAD

**Issue by JoeyG1973** [networktocode/ntc-templates#601]
##### ISSUE TYPE

 - Bug Report

##### TEMPLATE USING

```
Value VLAN_ID (\d+)
Value NAME (\S+)
Value STATUS (\S+)

Start
  ^\d+\s+enet\s+CE
  ^${VLAN_ID}\s+${NAME}\s+${STATUS} -> Record
```

##### SAMPLE COMMAND OUTPUT

```
'\nVLAN Name                             Status    Ports\n---- -------------------------------- --------- -------------------------------\n1    default                          active    Po1, Eth1/53, Eth1/54\n\nVLAN Type         Vlan-mode\n---- -----        ----------\n1    enet         CE     \n\nRemote SPAN VLANs\n-------------------------------------------------------------------------------\n\nPrimary  Secondary  Type             Ports\n-------  ---------  ---------------  -------------------------------------------\n\n\n\n'
```

##### SUMMARY

show vlan on nxos is not listing interfaces when parsed with platform cisco_nxos.  platform cisco_ios appears to work as expected  

##### STEPS TO REPRODUCE

```
>>> vlans = '\nVLAN Name                             Status    Ports\n---- -------------------------------- --------- -------------------------------\n1    default                          active    Po1, Eth1/53, Eth1/54\n\nVLAN Type         Vlan-mode\n---- -----        ----------\n1    enet         CE     \n\nRemote SPAN VLANs\n-------------------------------------------------------------------------------\n\nPrimary  Secondary  Type             Ports\n-------  ---------  ---------------  -------------------------------------------\n\n\n\n'
>>> parse_output(platform="cisco_nxos", command="show vlan", data=vlans)
[{'vlan_id': '1', 'name': 'default', 'status': 'active'}]

```

##### EXPECTED RESULTS

I expected to get the same results when using platform cisco_ios.
```
[{'vlan_id': '1', 'name': 'default', 'status': 'active', 'interfaces': ['Po1', 'Eth1/53', 'Eth1/54']}]
```
##### ACTUAL RESULTS

```
[{'vlan_id': '1', 'name': 'default', 'status': 'active'}]
```

**Comment by FragmentedPacket** [comment]
@JoeyG1973 This should be fixed with #604. Do you mind testing?

**Comment by JoeyG1973** [comment]
I was just about to dig in and fix this myself... let me grab the code and check it out
